.ado/templates/setup-repo.yml

```diff
@@ -0,0 +1,23 @@
+# This template install dependencies, builds, and bundles this repo
+steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: "12.x"
+    displayName: "Install Node.js"
```

Comment:
lower case "Install"?

Reply:
In regards to my comment in "azure-pipelines.yml", I wanted to lowercase only the scripts that run within our repo to make it easier to differentiate the steps when viewing results. In this case, NodeTool finds, downloads, and caches a version of Node and adds it to PATH on the agent machine. Since it's not a script within our repo, I thought I would leave it uppercase. Does that make sense? 

I know it's a little weird, but I was having trouble following the CI results in ADO and this just made it a bit easier.

Reply:
Correct me if I'm wrong, but it sounds like you're lowercasing build steps with repo specific scripts/content? I'm probably still not totally understanding but from the screenshot below, would we then want to lowercase "Kill FluentTester Process" or other repo specific steps like that then?

Fwiw this is super minor, but let's get on the same page for how we format our display names cross-plat.

![Screen Shot 2020-06-09 at 2 28 02 PM](https://user-images.githubusercontent.com/48299693/84202971-716ef900-aa5d-11ea-87d7-e155861add3c.png)

Reply:
"Correct me if I'm wrong, but it sounds like you're lowercasing build steps with repo specific scripts/**content**?"

Not the content part. The benchmark I was working off was:

Lowercase the displayName **iff** this script is executed **within** the repo.In other words,  is this script in a package.json within our repo? If so, lowercase. If not, uppercase

For the "Kill FluentTester", that is a PowerShell command to stop a process, not a specific command within the repo. This was the most helpful way for me looking at the results in Azure-pipelines. If you feel differently, I'm open to hear your thoughts.